code-challenges.js

```diff
@@ -2,7 +2,25 @@
 
 
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+const firstFib10 = () => {
+  // crate an array to store the numbers
+  let numArray = []
+  // create a 10x loop
+  for (i = 0; i < 10; i++) {
+    // put then number 1 into the i+1 position
+    if (i <= 1) {
+      numArray[i] = 1
+    } else {
+      // otherwise sum the previous 2 numbers
+      numArray[i] = numArray[i-1] + numArray[i-2]
+    }
+  }
 
+  // when all 10 positions are filled, return the array
+  return numArray
+}
```

Comment:
fabulous! I appreciate that you didn't just start the numArray with `[1,1]` and instead had the loop generate all the numbers. good work.